res/sections/65-Lez18042018.tex

```diff
@@ -77,19 +89,33 @@ \section{Target}
  \item \dots
 \end{itemize}
 
-In generale, è importante capire anche l'obiettivo dell'\textit{audience}: come
-mai il pubblico mi sta ascoltando? Che tipo di gergo devo usare? Un gerco
+In generale, è importante capire le competenze del pubblico e il suo obiettivo:
+come mai il pubblico mi sta ascoltando? Che tipo di gergo devo usare? Un gergo
 tecnico a uno molto semplice? Chi è qui cosa è interessato a sentire? Se il
 pubblico ha dei timori riguardo ad un certo argomento, è molto meglio
 rassicurarlo immediatamente: in questo modo gli si libererà la mente e sarà in
 grado di seguire meglio il discorso.
 
-Il \textbf{contesto} è importante. Dove quanto e in che situazione si parlerà?
+\section{Contesto}
+
+Il \textbf{contesto} in cui si tiene il discorso può influenzare l'esposizione.
+Dove, quanto e in che situazione si parlerà?
 Tutto ciò fa cambiare il modo in cui bisogna approcciarsi al pubblico.
-Presentare da soli è, per esempio, diverso da parlare in gruppo. In generale,
-il contesto influenza tutto il discorso e la sua durata.
 
-Finito tutto ciò, si pasa ai \textbf{contenuti} del discorso.
+Se si deve parlare davanti ad un piccolo gruppo di persone è più facile avere
+\emph{feedback} dal pubblico e cambiare di conseguenza il discorso, cosa invece
+molto più difficile se il pubblico è ampio. Un'altra cosa da valutare è se si è
```

Comment:
Ci son 4 ripetizioni della parola pubblico in 4 righe! Mettici qualche sinonimo ti prego :pray:

Reply:
no